src/main/webapp/app/entities/flair-bi/modal/modal-tabs/condition/condition.component.html

```diff
@@ -79,5 +93,6 @@
             </ui-select>
         </div>
     </div>
-    <condition-component show-add="true" show-delete="true" features="vm.features" condition="vm.condition.secondExpression" datasource-id="vm.datasourceId"></condition-component>
+    <condition-component show-add="true" show-delete="true" features="vm.features"
```

Comment:
if there is no change why this line is showing red...please change setting of ur editor...otherwise it is hard to indetify the changes